src/renderer/redux/stores/nodes/reducer.js

```diff
@@ -1,11 +1,22 @@
+import { deleteSubTree } from 'services/node';
+
 import { ACTIONS as APP_ACTIONS } from 'redux/stores/app/actions';
+import { ACTIONS } from './actions';
 
 export default (state = null, action) => {
   switch (action.type) {
     case APP_ACTIONS.REGISTER:
       return {};
     case APP_ACTIONS.LOGIN:
       return action.state || {};
+
+    case ACTIONS.CREATE:
+      return { ...state, ...action.node };
```

Comment:
why don't you put your the node in its id property ? (as you do with UPDATE/DELETE)

Reply:
oh I saw below that action.node would be `{myId: { ...myNode }}`

Reply:
Yup it's not really consistent actually but it's unit tested and all